src/Badges.sol

```diff
@@ -308,7 +311,44 @@ contract Badges is
       "mintWithConsent: invalid recipient signature"
     );
 
-    mint(_recipient, _uri, raftTokenId);
+    return mint(_recipient, _uri, raftTokenId);
+  }
+
+  function merkleMintWithConsent(
+    address _recipient,
+    address _issuer,
+    string calldata _uri,
+    bytes calldata _issuerSignature,
```

Comment:
one minor thing, the order for addresses in params is recipient and then issuer, but for signatures it is issuer then recipient. Just reads weird and I am concerned that this can easily be misread by devs. Can we change it pls?